core/block_svg.js

```diff
@@ -965,6 +977,12 @@ Blockly.BlockSvg.prototype.onMouseMove_ = function(e) {
       }
       this.setDragging_(true);
       this.moveToDragSurface_();
+
+
+      if (!Blockly.draggedConnectionManager_) {
+        Blockly.draggedConnectionManager_ =
```

Comment:
There is one draggedConnectionManager regardless of how many Blockly workspaces there are.  This is a small sanity check, but in reality the only way that the connectionManager would be set at this point in the code is if my touch/mouse code failed, and allowed two simultaneous drags.

The next step after I merge this will be to rebase my dragging branch on this, then start using this in Gesture.  Each Gesture will own one DraggedConnectionManager, and there will be no global one.

If you aren't comfortable with this hitting develop right now, I can make this into a new branch on google/blockly and do all of my dragging work on that.  My main concern was to avoid having a chance this size squashed in with the rest of my dragging code, but multiple pull requests on a non-develop branch would also solve that problem.